superset-frontend/src/explore/components/controls/DndColumnSelectControl/OptionWrapper.tsx

```diff
@@ -63,15 +63,11 @@ export default function OptionWrapper(
   const ref = useRef<HTMLDivElement>(null);
   const labelRef = useRef<HTMLDivElement>(null);
 
-  const item: OptionItemInterface = useMemo(
-    () => ({
-      dragIndex: index,
-      type,
-    }),
-    [index, type],
-  );
   const [{ isDragging }, drag] = useDrag({
-    item,
+    item: {
+      type,
+      dragIndex: index,
+    },
```

Comment:
It appears we're a few versions behind `react-dnd` (now on 14; we're on 11). Apparently the deps can now be given as a param after spec:

![image](https://user-images.githubusercontent.com/33317356/129902439-a5e22a8a-0aa3-4e0c-b9d8-baba38650710.png)

https://react-dnd.github.io/react-dnd/docs/api/use-drag

Might be a good TODO for when we bump this the next time.

Reply:
Good to know!